### PR TITLE
Add basic docker-compose config

### PR DESCRIPTION
**Pull request by timrourke**
This PR adds a very simple `docker-compose.yml` to make it easier to spin the application up locally.

### CODE REVIEW

**Comment by timrourke** [comment]
@cmvandrevala @paytonrules please review.

**Comment by timrourke** [comment]
@paytonrules Am I to understand that this change is also unacceptable?

File: docker/ruby/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+COPY Gemfile /app/Gemfile
+COPY Gemfile.lock /app/Gemfile.lock
+RUN bundle install
+
+# Copy in application's source
+COPY . /app
+
+# Apply migrations
+RUN bin/rake db:migrate
+RUN bin/rake db:seed
```
Comment (bartboy011) [inline docker/ruby/Dockerfile]: This command is intended to fill the DB with placeholder data... I'm not sure we need this run on every local build. Maybe we extract this to a separate bin script?

**Review by bartboy011 — CHANGES_REQUESTED** [review]
One thought:

File: docker/ruby/Dockerfile (hunk 1 above)
Comment (timrourke) [reply docker/ruby/Dockerfile]: yep, will change that.

**Pull request closed without merging**